<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphPropertiesWidget</class>
 <widget class="QWidget" name="GraphPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="KLineEdit" name="_graphName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Node Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="KColorCombo" name="_graphNodeColor"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Edge Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="KColorCombo" name="_graphEdgeColor"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="_graphOriented">
     <property name="text">
      <string>Oriented</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QCheckBox" name="_graphAutomate">
     <property name="text">
      <string>Automate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="KPushButton" name="_graphNodeColorApplyNow">
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="KPushButton" name="_graphEdgeColorApplyNow">
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="KPushButton" name="kpushbutton">
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
